autoware_adapi_v1_msgs/geometry/msg/Shape.msg

```diff
@@ -0,0 +1,18 @@
+# constants
+uint8 UNKNOWN = 0
+uint8 POINT3D = 1
+uint8 POINT2D = 2
+uint8 LINE3D = 3
+uint8 LINE2D = 4
+uint8 POLYGON3D = 5
+uint8 POLYGON2D = 6
+uint8 SPHERE = 7
+uint8 CIRCLE = 8
+uint8 MESH = 9
+uint8 PRISM = 10
+uint8 CYLINDER = 11
+
+# variables
+uint8 shape
+uint32[] sizes
+float32[] values
```

Comment:
I think value is too abstract.
It would be better to use something more meaningful, such as position value, size value.

Reply:
This field stores multiple data such as 2D-point, 3D-point, Z-range, and radius. Since the format changes depending on the shape, it cannot be given any more meaning.